src/app/core/http/users.service.ts

```diff
@@ -0,0 +1,42 @@
+import { StorageService } from './../services/storage.service';
+import { BehaviorSubject } from 'rxjs';
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+import { map } from 'rxjs/operators';
+import { environment } from 'src/environments/environment';
+import { RestService } from './rest.service';
+import { SessionState } from './authen.service';
+const notSignedInMessage = `Not signed in`;
+@Injectable({
+  providedIn: 'root'
+})
+export class UsersService extends RestService<IUser> {
+  public host: string;
```

Comment:
chỗ này em xóa đi, không cần khai báo host ở chỗ này, nó được kế thường từ RestService, nếu xài chỉ cần gọi this.host là được